Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.Events.cs

```diff
@@ -0,0 +1,142 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Windows.Foundation;
+using Windows.System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// The <see cref="ImageCropper"/> control allows user to crop image freely.
+    /// </summary>
+    public partial class ImageCropper
+    {
+        private void ControlButton_KeyDown(object sender, KeyRoutedEventArgs e)
+        {
+            var handled = false;
+            var diffPos = default(Point);
+            if (e.Key == VirtualKey.Left)
+            {
+                diffPos.X--;
+                handled = true;
+            }
+            else if (e.Key == VirtualKey.Right)
```

Comment:
I need to pull the PR down still to play with it first-hand, but how does this work with repeated strokes and holding both left and up for instance?